pydantic_ai_slim/pydantic_ai/_tool_manager.py

```diff
@@ -108,24 +108,21 @@ async def handle_call(
             raise ValueError('ToolManager has not been prepared for a run step yet')  # pragma: no cover
 
         if (tool := self.tools.get(call.tool_name)) and tool.tool_def.kind == 'output':
-            # Output tool calls are not traced and not counted
-            return await self._call_tool(
-                call,
-                allow_partial=allow_partial,
-                wrap_validation_errors=wrap_validation_errors,
-                approved=approved,
-            )
+            output_tool_flag = True
         else:
-            return await self._call_function_tool(
-                call,
-                allow_partial=allow_partial,
-                wrap_validation_errors=wrap_validation_errors,
-                approved=approved,
-                tracer=self.ctx.tracer,
-                include_content=self.ctx.trace_include_content,
-                instrumentation_version=self.ctx.instrumentation_version,
-                usage=self.ctx.usage,
-            )
+            output_tool_flag = False
+
+        return await self._call_function_tool(
```

Comment:
This name is no longer accurate, because we also use it for output tools now. Can we either merge the 2 methods (as there's no need for a separate `_call_tool` anymore), or call this method `_call_tool_traced`? The latter may be better to keep the concerns separated.